### PR TITLE
refactor resources routes into indv files by action

### PR DESCRIPTION
**Pull request by ashleygwilliams**
refactoring routes file to indvidual files and a manifest to prepare for adding joi validation and lout documentation to each endpoint.

### CODE REVIEW

**Comment by ashleygwilliams** [comment]
ready for review @sedge @gideonthomas :bow:

File: api/modules/files/routes/create.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+var controller = require('../controller');
+
+module.exports = [{
+    method: 'POST',
```
Comment (sedge) [inline api/modules/files/routes/create.js]: Was this jshinted? The indentation looks funny.

**Comment by gideonthomas** [comment]
r+

**Comment by sedge** [comment]
Other than a style nit this looks good. I would vote on squashing it all. Merge when ready!

**Pull request merged**